docs/api/app.md

```diff
@@ -501,6 +501,41 @@ gets emitted.
 **Note:** Extra command line arguments might be added by Chromium,
 such as `--original-process-start-time`.
 
+### Event: 'notification-activation' _macOS_ _Windows_
+
+Returns:
+
+* `event` Event
+* `appUserModelId` string - contains App module ID
```

Comment:
Hello @MarshallOfSound this is good remark, but I did forwarding parameters for this event according to list of those into INotificationActivationCallback::Activate https://docs.microsoft.com/en-us/windows/win32/api/notificationactivationcallback/nf-notificationactivationcallback-inotificationactivationcallback-activate

(It should to say that in this implementation I'm using it too but instead of static approach with Shortcut this activator creates dynamically into runtime not into compile time)

`HRESULT Activate(
  [in] LPCWSTR                            appUserModelId,
  [in] LPCWSTR                            invokedArgs,
  [in] const NOTIFICATION_USER_INPUT_DATA *data,
  [in] ULONG                              count
);`
appUserModelId is present. In my vision this parameter potentially can be used and for distinguish notifications between different AppId in bounds of one process, it is hard to imagine this case for me now but I have feeling that this possibility can be useful (so on given stage of codebase this parameter can be marked as for future use). 
I would to say - it is better to have more than less.

Reply:
> I would to say - it is better to have more than less.

I understand your reasoning but I think the _better_ thing to do would be to expose this instead as a `details` object with keys for `data`, `args` and then in the future if there is a use case for it exposing `appUserModelId`.  Currently having this prop exposed means this event can only ever be used on windows

Reply:
Hello @MarshallOfSound  this event is cross platform and works fine on Mac but for Mac appUserModelId holds notification identifier [notif.identifier UTF8String], that is also meaningful. Maybe it has sense to rename this parameter into identifier - in general appUserModelId is playing this role.